File: jung/src/main/java/RoadNetPA.java
public class RoadNetPA extends Core {

    private static final String PATH = "../samples/roadNet-PA.txt";
    private static final Boolean AVOID_DOUBLES = true;
    private static final Integer[] UNIQUE_TESTS = {100, 1000, 10000, 100000};
    private static final Integer[] INDEXES = {100, 1000, 10000, 100000};

    public static void main(String[] args) {
        run(PATH, AVOID_DOUBLES, UNIQUE_TESTS, INDEXES);
    }
}

File: quickgraph/RoadNetPA.cs
namespace quickgraph
{
	class RoadNetPA : Core
	{
		public RoadNetPA()
		{
			path = "../../../samples/roadNet-PA.txt";
			avoidDoubles = true;
			uniqueTests = new int[] { 100, 1000, 10000 };
			indexes = new int[] { 100, 1000, 10000, 100000 };
			algorithms = new string[] { "bfs", "dfs", "shortest" };
		}
	}
}